webcompat/templates/layout.html

```diff
@@ -46,3 +46,4 @@
 {% block extrascripts %}{% endblock %}
 </body>
 </html>
+</html>
```

Comment:
Hmm this doesn't seem to be a legit thing 😜

Reply:
hehe
Sorry for that!